rollup.config.js

```diff
@@ -0,0 +1,36 @@
+import babel from '@rollup/plugin-babel'
+import commonjs from '@rollup/plugin-commonjs'
+import resolve from '@rollup/plugin-node-resolve'
+import typescript from 'rollup-plugin-typescript2'
+import BabelLoaderExcludeNodeModulesExcept from 'babel-loader-exclude-node-modules-except'
+
+import { DEFAULT_EXTENSIONS } from '@babel/core'
+const extensions = [...DEFAULT_EXTENSIONS, '.ts', '.tsx']
+
+const packageJson = require("./package.json");
+
+export default {
+	input: 'lib/index.ts',
+	output: [
+		{
+			file: 'dist/bundled/index.cjs.js',
```

Comment:
this doesn't match the package.json module path – intended?

Reply:
I was thinking about still providing the plain non-transpiled ones, but apparently code-deduplication will only work if the exact versions match (e.g. for core-js/semver) so it is probably not worth the additional complexity. I did some comparison and there is not much saving in space between the babel-only and the bundled packages.